gcc/config/aarch64/winnt.cc

```diff
@@ -1125,8 +1125,9 @@ aarch64_pe_file_end (void)
 	    continue;
 	  name += 7;
 	  fprintf (asm_out_file, "\t.section\t.rdata$%s, \"dr\"\n"
-	  		   "\t.globl\t%s\n"
-			   "\t.linkonce\tdiscard\n", oname, oname);
+			"\t.globl\t%s\n"
```

Comment:
The indentation style of the former code was not consistent either but looking at the file, the majority seems to use spaces instead of tabs.

Reply:
Indentation in the GNU toolchain is using two spaces for indent but consolidating 8 spaces into a tab. 
With those settings in the editor, I think the tabulation of fprintf parameters is correct how I have it.
Parameter indentation does not seem to have a specific rule. Just parameters to look defined.